### DISCUSSION TITLE
pimcore 6 bundle not recognized in pimcore x?

### DISCUSSION THREAD

**Discussion by Rojal123245** [pimcore/pimcore#9299] [Q&A / Support]
i like to install my pimcore 6 bundle in pimcore x but after i added all bundle in src it didn't recognized it ?

**Comment by dvesh3** [comment]
make sure you register the bundle namespace in the "autoload:psr-4" section e.g. "TestBundle\\": "src/TestBundle" and then run `composer dump-autoload -o`
